### PR TITLE
Richard Severich the Kata 2 Sort Inner Content.

### PR DESCRIPTION
**Pull request by RichardSeverich**
This is my first commit of the Kata 2 Sort Inner Content exercise.

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    }
+
+    /**
+     * This method rolls down a querevide string as a parameter.
+     *
+     * @param string This parameter is the string that will be sorted in descending order.
+     * @return A sorted string is returned in descending order.
+     */
+    private String getDescendingOrder(String string) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java]: Try to use an existing function for this

File: src/test/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContentTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        String expectedResult = "tihs ktaa is esay";
+        assertEquals(expectedResult, sortInner.sortInner(firstCad));
+    }
+
+    /**
+     * This test checks the string that returns the method getSortInner returns the expected string.
+     */
+    @Test
+    public void verifyChangeOfSortTheInnerContentInDescendingOrder() {
+        SortInnerContent sortInner = new SortInnerContent();
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContentTest.java]: You are repeating this code in all your tests.
Please put them in @before method or convert it to a static method.

File: src/test/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContentTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by RichardSeverich on 5/23/2017.
+ */
+public class SortInnerContentTest {
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContentTest.java]: I found a bug in your implementation
What should happen in this scenario?

```java
 assertEquals("I am", sortInnerContent.sort("I am"));
```

Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContentTest.java]: Hi, ok I'll check it out.

File: src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+     */
+
+    public String sortInner(String cad) {
+
+        String[] partsWords;
+        partsWords = cad.split(" ");
+        String cadDescending;
+        StringJoiner stringJoiner = new StringJoiner(" ");
+        String word;
+        for (int i = 0; i < partsWords.length; i++) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java]: you can use a forEach here

File: src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+            if (partsWords.length < THREE) {
+                word = partsWords[i];
+            } else {
+                cadDescending = partsWords[i].substring(1, partsWords[i].length() - 1);
+                cadDescending = Stream.of(cadDescending.split("")).sorted(Comparator.reverseOrder())
+                        .collect(Collectors.joining());
+                word = String.join("", String.valueOf(partsWords[i].charAt(0)), cadDescending,
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java]: You don't need to use a String.valueOf here

File: src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                        String.valueOf(partsWords[i].charAt(partsWords[i].length() - 1)));
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Richard_Severich_Sort_Inner_Content/SortInnerContent.java]: Same as above

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**